docs/_sass/minima/_base.scss

```diff
@@ -271,11 +271,24 @@ table {
   }
 }
 
-/**
- * Prevents page break to cut through content when converting to pdf
- */
 @media print {
+  /**
+  * Prevents page break to cut through content when converting to pdf
+  */
   body {
     display: block;
   }
+  /**
+  * Replaces default site header (with navigation) with a generic header name (AB-3) when converting to pdf.
+  */
+  .site-header .wrapper {
+    display: none;
+  }
+  .site-header {
+    text-align: center;
+  }
+  .site-header:before {
+    content: "AB-3";
+    font-size: 32px;
+  }
```

Comment:
How about this as the commit message?
```
Omit the top navigation menu when a page is printed

When a page in the product website (e.g, the User Guide) is converted
into a print view, the top navigation bar appears in printed version.
This is not appropriate as a navigation bar belongs in a Web page but
not in a printed view. This is particularly a problem when documents
in the website are converted into PDF, which is done by 'printing' the
page into a PDF file.

Let's update the behaviour of the media print query to replace the top
navigation menu with simply the project name.
```

Reply:
Looks much clearer than my previous one, I'll amend the commit message and comments as stated above.